cmd/device-worker/main.go

```diff
@@ -115,3 +108,37 @@ func main() {
 		log.Fatal("Cannot start worker server, err:", err)
 	}
 }
+
+var (
```

Comment:
Could you move the variables definition before the main ?
We don't have any convention for that (nor we don't use formatter), but it feels more convenient to see the variable definition before they are being used.

Reply:
I left it as it was.